### DISCUSSION TITLE
Supported python version

### DISCUSSION THREAD

**Discussion by erdies** [falkTX/Carla#1947] [General]
Hi,
carla is great, I really like it. I am using Gentoo Linux and Carla was included in the gentoo main repo but has been recently removed and is therefore available in certain overlays. The issue is Gentoo is moving to higher python version, my main version is python-2.12.7. 
All Carla Overlay versions seem to rely on lower python versions. The consequence is, that you are going to install multiple python version on your system once you try to add Carla to your system. I have tested Carla on my computer and it builds and runs flawless with python-2.12.7
So it would be helpfull if you could confirm and announce to support these higher version officially. My hopy is that the gentoo guys will then include carla in the main gentoo repo again. I could even try to contact them to trigger further support for Carla and adjust their ebuild. This would be a great enrichment for the gentoo community.

Thanks a lot
Martin

**Comment by falkTX** [comment]
there is no support for python2 since many years? why still using python2? this is not a higher version but something that has been "end of life" since quite a while now.
